go.mod

```diff
@@ -13,28 +13,28 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
 	github.com/go-openapi/spec v0.19.4
-	github.com/gogo/protobuf v1.3.1
+	github.com/gogo/protobuf v1.3.1 // indirect
 	github.com/golang/mock v1.3.1
+	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/mailgun/mailgun-go/v3 v3.6.0
-	github.com/operator-framework/operator-sdk v0.15.1
-	github.com/pborman/uuid v1.2.0
+	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
```

Comment:
what happened with these?

Reply:
and also this one? 
better to keep it untouched.

Reply:
Updated, adding only openshift API modules